src/Logic/Utilities.cs

```diff
@@ -3069,6 +3069,17 @@ public static string RemoveUnneededSpaces(string text, string language)
             if (text.Contains("<u>", StringComparison.OrdinalIgnoreCase) && text.Contains("</u>", StringComparison.OrdinalIgnoreCase))
                 text = RemoveSpaceBeforeAfterTag(text, "<u>");
 
+            // Font
+            if (text.Contains("<font ", StringComparison.OrdinalIgnoreCase))
+            {
+                var idx = text.IndexOf("<font ", StringComparison.OrdinalIgnoreCase);
+                var endIdx = text.IndexOf('>', idx);
+                if (endIdx > idx && endIdx < text.Length)
+                {
+                    var color = text.Substring(idx, (endIdx - idx) + 1);
+                    text = RemoveSpaceBeforeAfterTag(text, color);
+                }
+            }
             text = text.Trim();

```

Comment:
Perhaps, it might be more efficient to combine `.Contains()` and `.IndexOf()`?
The `endIdx < text.Length` expression looks like an 'always true' condition?
```
{
    var idx = text.IndexOf("<font ", StringComparison.OrdinalIgnoreCase);
    if (idx >= 0)
    {
        var endIdx = text.IndexOf('>', idx);
        if (endIdx > idx) // or (endIdx > 0)
        {
            ...
        }
    }
}
```